UnicodeDecodeError: 'cp950' codec can't decode byte 0xe2 in position 1068: illegal multibyte sequence

```sh
pytest
```

```sh
tests/cz_gitmoji/test_cz_gitmoji.py::test_info FAILED                                                                                                                                         [ 43%]

============================================================================================= FAILURES ============================================================================================= 
____________________________________________________________________________________________ test_info _____________________________________________________________________________________________ 

cz_gitmoji = <cz_gitmoji.main.CommitizenGitmojiCz object at 0x0000022A60476120>

    def test_info(cz_gitmoji: CommitizenGitmojiCz) -> None:
        """Verify the info content."""
>       info = cz_gitmoji.info()

tests\cz_gitmoji\test_cz_gitmoji.py:86:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  

self = <cz_gitmoji.main.CommitizenGitmojiCz object at 0x0000022A60476120>

    def info(self) -> str:
        """Return information about the commit message style."""
        dir_path = Path(__file__).parent
        # TODO: improve the info file
        filepath = dir_path.joinpath("conventional_gitmojis_info.txt")
        with open(filepath, "r") as f:
>           content = f.read()
E           UnicodeDecodeError: 'cp950' codec can't decode byte 0xe2 in position 1068: illegal multibyte sequence
```

https://github.com/ljnsn/cz-conventional-gitmoji/blob/b26afbedfb3aa381da02c40bc30c6400c7927110/src/cz_gitmoji/main.py#L322

It seems that line is missing `encoding="utf-8"`.

Actually, I've encountered similar issues not only with pytest but also in other places on Windows.

--- a/src/cz_gitmoji/main.py
+++ b/src/cz_gitmoji/main.py
@@ -319,7 +319,7 @@ class CommitizenGitmojiCz(BaseCommitizen):
         dir_path = Path(__file__).parent
         # TODO: improve the info file
         filepath = dir_path.joinpath("conventional_gitmojis_info.txt")
-        with open(filepath, "r", encoding="utf-8") as f:
+        with filepath.open(encoding="utf-8") as f:
             content = f.read()
         return content
 

--- a/src/cz_gitmoji/main.py
+++ b/src/cz_gitmoji/main.py
@@ -319,7 +319,7 @@ class CommitizenGitmojiCz(BaseCommitizen):
         dir_path = Path(__file__).parent
         # TODO: improve the info file
         filepath = dir_path.joinpath("conventional_gitmojis_info.txt")
-        with open(filepath, "r") as f:
+        with open(filepath, "r", encoding="utf-8") as f:
             content = f.read()
         return content
 

